k8s/vagrant/README.md

```diff
@@ -49,4 +58,69 @@ Just use *vagrant up* to bring up the cluster.
 ubuntu-host:~/k8s-demo$ vagrant up
 ```
 
+### Verify
+
+You should see output similar to this:
```

Comment:
The output displayed will be similar to the following:

Reply:
done